generatemanmenu.pl

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+# Output to stdout
+my $OutManMenu = \*STDOUT;
+
+# Make option menu current with most recent version of curl.
+# Output options to file, remove first line, and make page for option menu.
+my $curlPath = @ARGV[0];                    # store path to curl in variable
+`$curlPath/src/curl -h all > options.txt`;  # output options to text file
+`tail -n +2 options.txt > tempOptions.txt`; # remove first line, outputting to tempOptions.txt
+`mv tempOptions.txt options.txt`;           # move temp to file options.txt        
+
+# Store option file in variable
+my $_options = "options.txt"; 
+
+# Output opening html tags to stdout
+print  $OutManMenu <<OPENMANMENUHTML
+<!-- Option Search Menu -->
+<div id="manpage-option-menu" class="sideStay sideStayLeft inactive" 
+ data-to-side="1" data-on-side="0"> 
+<!-- On Off button -->
+  <button class="searchButtonToSide" id="searchButton" 
+   onclick="toggleSearchButton(this.nextElementSibling, this)" data-onoff="0"
+   onmouseenter="buttonToSide(1, this.dataset.onoff)" data-to-Side="1"
+   onmouseleave="buttonToSide(0, this.dataset.onoff)">Show Options</button><br>
+  <div style="display:none">
+    <div class="searchBox">
+      <label for="optionSearchField">Search Options</label><br>
+      <img src="https://curl.se/logo/curl-symbol.svg" alt="curl symbol">
+      <input id="optionSearchField" name="optionSearchField" placeholder="search options"
+       type="search" oninput="searchOptions(this.value)">
+    </div>
+    <ul id="optionMenu">
+      <!-- Start option list -->
+      <div id="fullOptionList" style="display:none">
+OPENMANMENUHTML
+;
```

Comment:
Wouldn't it be nicer to have the HTML file in a template file to make it easier to edit separately from the script logic? That's how we usually make these things.

Reply:
Will do.